src/harshad/selenium_assignment3/SelectDropDownDemoWithFacebookSignUp.java

```diff
@@ -0,0 +1,171 @@
+			newPassword.sendKeys("abc123456");
+			System.out.println("Password entered");
+			
+			birthDate = driver.findElement(By.id("day"));
+			Select daySelect = new Select(birthDate);
+			daySelect.selectByValue("5");
+			
+			birthMonth = driver.findElement(By.id("month"));
+			Select monthSelect = new Select(birthMonth);
+			monthSelect.selectByVisibleText("Apr");
+			
+			birthYear = driver.findElement(By.id("year"));
+			Select yearSelect = new Select(birthYear);
+			yearSelect.selectByValue("1995");
+			
+			System.out.println("Date of Birth selected from dropdown is: "+birthDate.getAttribute("value")+" " +birthMonth.getAttribute("value").toString()+ " "+ birthYear.getAttribute("value"));
+			
+			gender = driver.findElement(By.id("u_0_5"));
+			gender.click();
+			if(gender.getAttribute("value").equals("1")) {
+				System.out.println("Gender selected as Female");
+			}else if(gender.getAttribute("value").equals("2")) {
+				System.out.println("Gender selected as Male");
+			}else {
+				System.out.println("Gender selected as Custom");
+			}
+			
+			signUpBtn = driver.findElement(By.name("websubmit"));
+			signUpBtn.click();
+			System.out.println("SignUp button is clicked");
+		}else {
+			System.out.println("Verified title of current page is not �Sign up for Facebook | Facebook� it is "+driver.getTitle());
+		}
+		
+		try {
+			Thread.sleep(5000);
+		} catch (InterruptedException e) {
+			e.printStackTrace();
+		}
+		
+		return driver;
+	}*/
+	
+	//This method will perform steps by finding elements By.xpath
+	private static WebDriver openBrowserAndPerformStepsUsingXpath(String url) {
+		
+		WebDriver driver = new ChromeDriver();
+		driver.navigate().to(url);
+		System.out.println("Create new account page of facebook is successfully opened");
+		if(driver.getTitle().equals("Sign up for Facebook | Facebook")) {
+			System.out.println("Verified page title is �Sign up for Facebook | Facebook�");
+			
+			
+			firstName = driver.findElement(By.xpath("//input[@name=\"firstname\"]"));
+			firstName.sendKeys("Harshad");
+			System.out.println("FirstName entered as: "+firstName.getAttribute("value"));
+			
+			surname = driver.findElement(By.xpath("//input[@name=\"lastname\"]"));
+			surname.sendKeys("Bhore");
+			System.out.println("Surname entered as: "+surname.getAttribute("value"));
+			
+			emailIDOrMobile = driver.findElement(By.xpath("//input[@name=\"reg_email__\"]"));
+			emailIDOrMobile.sendKeys("788898989hjhj88");
+			System.out.println("Email ID or Mobile number entered as: "+emailIDOrMobile.getAttribute("value"));
+			
+			newPassword = driver.findElement(By.xpath("//input[@name=\"reg_passwd__\"]"));
+			newPassword.sendKeys("abc123456");
+			System.out.println("Password entered");
+			
+			birthDate = driver.findElement(By.xpath("//select[@id=\"day\"]"));
+			Select daySelect = new Select(birthDate);
+			daySelect.selectByValue("5");
+			
+			birthMonth = driver.findElement(By.xpath("//select[@id=\"month\"]"));
```

Comment:
Please do not use \ and " " for xpath.
Always use single inverted comma. ' ' for attribute value.